python/src/trezorlib/_internal/emulator.py

```diff
@@ -154,6 +167,8 @@ def launch_process(self) -> subprocess.Popen:
         )
 
     def start(self) -> None:
+        global _RUNNING_PIDS
```

Comment:
in none of these places `_RUNNING_PIDS` needs to be global

Reply:
57ae7199f1